### PR TITLE
[TSAtds] Add PGI non-CUDA + update PrgEnv-*

### PR DESCRIPTION
**Pull request by kraushm**
This PR will add PGI 19.9 noCUDA, update the production list for Tsa RH7.7, and update PrgEnv* easyconfig files.

It should be merged after PR https://github.com/eth-cscs/production/pull/1513, since it builds on modifications of the Machine file.

### CODE REVIEW

**Comment by kraushm** [comment]
retest this please

**Comment by kraushm** [comment]
The Jenkins check fails for the newly added PrgEnv-pgi/gnu-nocuda modules, since the folders/modules invoked by the PrgEnv(s) do not exist yet.

File: easybuild/easyconfigs/o/OpenMPI/OpenMPI-4.0.2-GCC-8.3.0.eb [hunk 1]
```diff
@@ -26,6 +26,7 @@ configopts = (
     "--without-alps "
     "--without-libfabric "
     "--without-lsf "
+    "--without-mxm "
```
Comment (gppezzi) [inline easybuild/easyconfigs/o/OpenMPI/OpenMPI-4.0.2-GCC-8.3.0.eb]: you are modifying an existing eb recipe, can you confirm that this is not currently deployed in (pre-)production?

File: easybuild/easyconfigs/p/PrgEnv-gnu/PrgEnv-gnu-tsa-19.2.eb [hunk 2]
```diff
@@ -15,7 +15,7 @@ description = """
 
 toolchain = SYSTEM
 
-local_path = '/apps/arolla/UES/jenkins/RH7.6/%s/%s/easybuild/modules/all' % (local_prgenv, version)
+local_path = '/apps/arolla/UES/jenkins/RH7.7/%s/%s/easybuild/modules/all' % (local_prgenv, version)
```
Comment (gppezzi) [inline easybuild/easyconfigs/p/PrgEnv-gnu/PrgEnv-gnu-tsa-19.2.eb]: same here, are you sure this has no side-effects in the current deployed system?

**Comment by kraushm** [comment]
retest this please

File: easybuild/easyconfigs/p/PrgEnv-gnu/PrgEnv-gnu-tsa-19.2.eb (hunk 2 above)
Comment (kraushm) [reply easybuild/easyconfigs/p/PrgEnv-gnu/PrgEnv-gnu-tsa-19.2.eb]: This one was reverted to it's original and a new module (with suffix RH7.7) was added.

File: easybuild/easyconfigs/o/OpenMPI/OpenMPI-4.0.2-GCC-8.3.0.eb (hunk 1 above)
Comment (kraushm) [reply easybuild/easyconfigs/o/OpenMPI/OpenMPI-4.0.2-GCC-8.3.0.eb]: This eb recipe was present in master, but in the end was not deployed on the system.

**Pull request merged**